fizzbuzz_spec.py

```diff
@@ -1,13 +1,20 @@
 # -*- coding: utf-8 -*-
 
 from expects import *
+from fizzbuzz import *
 
-
+results = {
+    1: 1,
+    2: 2,
+    3: "Fizz",
+    5: "Buzz",
+   15: "FizzBuzz",
+}
+ 
 with describe('Fizzbuzz'):
```

Comment:
Hello and thanks for the interest :-)

The problem that I personally see to this kind of test, it's that you don't see which is the business logic applied, which are the requirements being solved: why 3 is "Fizz", why 5 is "Buzz"? The tests should be a main part of your documentation.

IMHO, this kind of tests (something like a parametrized tests) might be good for a final stage, where you don't want to have 100 tests, one for each use case: once you have already discovered and solved the business logic, then it might be good to have a kind of "overall" test to complement it, where you see many other cases. 

Indeed, I tried that approach (it's one of the repos that we include in the README) :-)